src/app/features/search/components/search-filter/search-filter.component.ts

```diff
@@ -2,12 +2,29 @@ import { NgClass } from '@angular/common';
 import { ChangeDetectionStrategy, ChangeDetectorRef, Component, inject, input, OnInit } from '@angular/core';
 import { DepartureDateWithIds } from '@features/search/interfaces/filter-dates.interface';
 import { SearchFilterService } from '@features/search/services/search-filter/search-filter.service';
-import { dateConverter2 } from '@shared/utils/date-converter';
+import { convertToDate } from '@shared/utils/convertToDateWithTime';
 import { TuiItem } from '@taiga-ui/cdk';
 import { TuiButton, TuiLoader } from '@taiga-ui/core';
 import { TuiCarouselButtons, TuiCarouselComponent } from '@taiga-ui/kit';
 import { fromEvent, Observable, Subscription } from 'rxjs';
 
+const mobileWidth = 520;
+const middleWidth = 650;
+const tabletWidth = 800;
+
+function getItemsCountFromWidth(width: number) {
+  if (width < mobileWidth) {
+    return 2;
+  }
+  if (width < middleWidth) {
+    return 3;
+  }
+  if (width < tabletWidth) {
+    return 4;
+  }
+  return 5;
```

Comment:
Magic numbers

Reply:
my function is called getItemsCountFromWidth, so 2, 3, 4, 5 are ItemsCount

Reply:
What kind of items?

Reply:
That's another question :smile: